tests/constant_tidal_heating.prm

```diff
@@ -0,0 +1,115 @@
+#This parameter file tests constant tidal heating.
+#
+#Heating equation is from Tobie et al. (2003) (https://doi.org/10.1029/2003JE002099)
+#H= 2*(viscosity)*(time-averaged strain rate)^2/(1+((viscosity)*(tidal frequency)/(shear modulus))^2))
+#viscosity = constant_viscosity
+#time-averaged strain rate = constant_tidal_strain_rate
+#tidal frequency = constant_tidal_frequency
+#shear modulus = constant_shear_modulus
+#
+#The equation in the model is simplized as (density)*(specific heat capacity)*dT/dt=H, as unit of H is W/m^3.
+#It shows temperature increase with time in a domain without conduction and convection.
+#
+#Model calculates temperature increase from 100 K for 1 Myr.
+#analytical temperature increase over is 94.19673477 K.
+#It matches with numerical model result, having temperature increase by 94.196735 K.
+
+
+set Dimension                              = 2
+set Use years in output instead of seconds = true
+set End time                               = 1e6
+
+set Output directory                       = constant_tidal_heating
```

Comment:
I think we should still keep the name of the PRM and test output folder as constant_tidal_heating